src/fairchem/core/common/gp_utils.py

```diff
@@ -282,10 +284,12 @@ def reduce_from_model_parallel_region(input: torch.Tensor) -> torch.Tensor:
 def scatter_to_model_parallel_region(
     input: torch.Tensor, dim: int = -1
 ) -> torch.Tensor:
+    assert initialized(), "Cannot use graph parallel with initializing gp group, must call setup_gp from gp_utils.py!"
```

Comment:
This is great, but can you add the same assert to `copy_to_model_parallel_region` and `reduce_from_model_parallel_region` functions as well for consistency.

Reply:
yes im planning to remove it, just leave it here for now to do some sanity checks